components/BottomSheet/examples/BottomSheetTypicalUseExample.m

```diff
@@ -0,0 +1,120 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialBottomSheet.h"
+#import "MaterialButtons.h"
+
+@interface ExampleScrollView : UIScrollView
+@end
+
+@implementation ExampleScrollView
+@end
+
+@interface BottomSheetExampleViewController : UIViewController
+@end
+
+@implementation BottomSheetExampleViewController
+
+- (void)loadView {
+  UIScrollView *scrollView = [[ExampleScrollView alloc] initWithFrame:CGRectZero];
+  scrollView.contentSize = CGSizeMake(100, 1000);
+  self.view = scrollView;
+  self.view.backgroundColor = [UIColor redColor];
```

Comment:
Can you make this a gradient or a tall textfield so we can verify the scrolling behavior?

I'm concerned the presented scrollview may conflict with the drag down to dismiss behavior of the presentation controller.